superset/assets/src/explore/constants.js

```diff
@@ -57,3 +57,11 @@ export const sqlaAutoGeneratedMetricRegex = /^(LONG|DOUBLE|FLOAT)?(SUM|AVG|MAX|M
 export const druidAutoGeneratedMetricRegex = /^(LONG|DOUBLE|FLOAT)?(SUM|MAX|MIN|COUNT)\([A-Z0-9_."]*\)$/i;
 
 export const EXPLORE_ONLY_VIZ_TYPE = ['separator', 'markup'];
+
+export const TIME_FILTER_LABELS = {
+  time_range: 'Time Range',
```

Comment:
I think these should be wrapped in `t()` now

Reply:
fixed!